src/frameworks/docs/flutter.md

```diff
@@ -0,0 +1,46 @@
+Project: /docs/hosting/_project.yaml
+Book: /docs/_book.yaml
+page_type: guide
+
+{% include "_shared/apis/console/_local_variables.html" %}
+{% include "_local_variables.html" %}
+{% include "docs/hosting/_local_variables.html" %}
+
+<link rel="stylesheet" type="text/css" href="/styles/docs.css" />
+
+# Integrate Flutter Web
+
+With the Firebase framework-aware {{cli}}, you can deploy your Flutter application
+to Firebase and serve dynamic content to your users.
+
+<<_includes/_preview-disclaimer.md>>
+
+<<_includes/_before-you-begin.md>>
+
+<<_includes/_initialize-firebase.md>>
+
+1. Choose your hosting source directory; this could be an existing Flutter app.
+1. Choose "Dynamic web hosting with web framework."
```

Comment:
This is a bug in the existing docs.
1. It's not "Dynamic web hosting with web frameworks" in the CLI. The prompt is "Do you want to use a web framework (experimental)"
2. We ask for a directory after prompting whether to use web frameworks